android/quest/src/main/java/org/smartregister/fhircore/quest/ui/register/patients/RegisterViewModel.kt

```diff
@@ -711,34 +711,41 @@ constructor(
       }
 
       val todayCases = patients.filter {
-
-        if(it.extension?.get(0)?.value?.asStringValue()?.isNotEmpty() == true){
-          val date = convertToDateStringToDate(it.extension?.get(0)?.value?.asStringValue() ?: "")
-          date.isToday()
+        val extension = it?.extension?.find { it.url?.substringAfterLast("/").equals("patient-registraion-date") }
+        if(extension != null && extension.value?.asStringValue()?.isNotEmpty() == true){
+          val date = convertToDateStringToDate(extension.value?.asStringValue())
+          date?.isToday() ?: false
         }else{
-          it.meta?.lastUpdated?.isToday() == true
+          false
```

Comment:
what does it mean to return false?

Reply:
If patient does not have the registration date then I was using the "lastUpdated" which may have wrong date. Now I am not counting patients which doesn't have the registration date because now all patients must have a registration date.